v20.1/upgrade-cockroach-version.md

```diff
@@ -259,7 +267,7 @@ Once you are satisfied with the new version:
     > RESET CLUSTER SETTING cluster.preserve_downgrade_option;
     ~~~
 
-3. Update your application or tooling to re-enable schema changes that were disallowed during the upgrade process. **Note that this applies only to upgrades from 19.2 to 20.1. It does not apply to other upgrades.**
+3. Update your application or tooling to re-enable schema changes that were disallowed during the upgrade process. Note that this applies only to upgrades from 19.2 to 20.1. It does not apply to other upgrades.
```

Comment:
Change `19.2 to 20.1` to `v19.2 to v20.1`

Reply:
Done.